ui/pages/onboarding-flow/privacy-settings/privacy-settings.js

```diff
@@ -242,16 +242,17 @@ export default function PrivacySettings() {
                     }}
                   />
                   {ipfsURL ? (
-                    <Typography
-                      variant={TypographyVariant.H7}
+                    <Text
+                      variant={TextVariant.bodySm}
+                      as="h6"
```

Comment:
So I've started to see `as="h6"` popping up in new components and I've realized by updating the html element to `h6` during these updates we've unintentionally been promoting bad sematic HTML. Let's try to use better html elements and update the e2e tests instead of the elements moving forward. It's rare that we would want to use an `h6` tag I think.
```suggestion
```